impedance/validation.py

```diff
@@ -104,38 +103,39 @@ def get_tc_distribution(f, M):
             p_values, mu = fitLinKK(f, ts, M, Z)
 
             if M % 10 == 0:
-                print(M, mu, rmse(eval_linKK(p_values, ts, f), Z))
+                print(M, mu, rmse(eval_linKK(p_values, R0, ts, f), Z))
     else:
         M = max_M
         ts = get_tc_distribution(f, M)
         p_values, mu = fitLinKK(f, ts, M, Z)
 
-    return M, mu, eval_linKK(p_values, ts, f), \
-        residuals_linKK(p_values, ts, Z, f, residuals='real'), \
-        residuals_linKK(p_values, ts, Z, f, residuals='imag')
+    return M, mu, eval_linKK(p_values, R0, ts, f), \
+        residuals_linKK(p_values, R0, ts, Z, f, residuals='real'), \
+        residuals_linKK(p_values, R0, ts, Z, f, residuals='imag')
 
 
 def fitLinKK(f, ts, M, Z):
     """ Fits the linKK model using scipy.optimize.least_squares """
-    initial_guess = np.append(min(np.real(Z)),
-                              np.ones(shape=(M,)) *
-                              ((max(np.real(Z))-min(np.real(Z)))/M))
+    R0 = min(np.real(Z))
+    initial_guess = np.ones(shape=(M,)) * \
+                           ((max(np.real(Z))-min(np.real(Z)))/M)
 
     result = least_squares(residuals_linKK, initial_guess, method='lm',
-                           args=(ts, Z, f, 'both'),
+                           args=(R0, ts, Z, f, 'both'),
                            ftol=1E-13, gtol=1E-10)
 
     p_values = result['x']
-    mu = calc_mu(p_values[1:])
+    mu = calc_mu(p_values)
 
     return p_values, mu
 
 
-def eval_linKK(Rs, ts, f):
+def eval_linKK(Rs, R0, ts, f):
```

Comment:
Great job breaking out R0 into a constant instead of a fit parameter 👑

Reply:
Thanks! I looked at trying to use the approach for holding constants in circuit models, but this seemed more straight forward.